proto-google-cloud-bigtable-admin-v2/clirr-ignored-differences.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please revert the clirr changes

Reply:
I prefer @ad548's approach of silencing method additions to autogenerated protos. We dont consider the protos as part of the public api and furthermore we definitely don't support users implementing autogenerate proto interfaces

Reply:
they are already added, this is just a merge conflict so they can be safely removed :)

Reply:
Removed these changes.